api/controller/xtra_db_cluster_api.proto

```diff
@@ -88,8 +88,8 @@ message CreateXtraDBClusterRequest {
       msg_exists: true
     }
   ];
-  // PMM server public address. If empty do not turn on monitoring.
-  string pmm_public_address = 4;
```

Comment:
i have a question.
will deleting a field break anything with the previous releases? I was under the impression that we don't ever change any fields and if we want to stop using them we should just add a deprecate.

Reply:
You are right, we should do that in public APIs, but it's private API and wasn't published yet and used only to communicate between dbaas-controller and pmm-managed which updates in the same time.